contrib/buildahimage/upstream/Containerfile

```diff
@@ -35,21 +38,35 @@ RUN useradd build; yum -y update;  rpm --restore shadow-utils 2>/dev/null; yum -
      runc \
      fuse-overlayfs \
      fuse3 \
-     containers-common; \
+     containers-common --exclude container-selinux; \
```

Comment:
why are we excluding c-selinux?

Reply:
NVM, I see, we missed adding that in this containerfile, we have it in the others.